Pod/Assets/Scripts/iconVersioning.sh

```diff
@@ -67,7 +67,7 @@ function processIcon() {
     base_file=$1
 
     cd "${CONFIGURATION_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}"
-    base_path=`find . -name ${base_file}`
+    base_path=`find . -maxdepth 2 -name ${base_file}`
```

Comment:
This can be replaced by:

    base_path=`find . -name ${base_file} -print -quit`

which is less hacky than this `2` depth limit.